test/scripts/openshift-ci/install-operator.sh

```
@@ -0,0 +1,404 @@
+    local detected_channel detected_csv
+    detected_channel=$(echo "${info}" | awk '{print $1}')
+    detected_csv=$(echo "${info}" | awk '{print $2}')
+    echo "  Catalog reports: channel=${detected_channel} csv=${detected_csv}"
+    OPERATOR_CHANNEL="${detected_channel}"
+
+    if [[ -n "${OPERATOR_VERSION}" && -n "${csv_pattern}" && "${detected_csv}" != "${csv_pattern}" ]]; then
+        echo "  ERROR: requested ${csv_pattern} but catalog only offers ${detected_csv}"
+        echo "  Check that OPERATOR_VERSION=${OPERATOR_VERSION} is available in the catalog."
+        exit 1
+    elif [[ -z "${OPERATOR_VERSION}" && -n "${csv_pattern}" && -n "${detected_csv}" ]]; then
+        CSV_VERSION="${detected_csv#${OPERATOR_NAME}.}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)